ha_mroonga.cpp

```diff
@@ -1868,6 +1939,7 @@ static int mrn_init(void *p)
   }
 err_log_file_open:
   mysql_mutex_destroy(&mrn_log_mutex);
+  mysql_mutex_destroy(&mrn_query_log_mutex);
```

Comment:
`mrn_query_log_mutex` isn't destroyed when `mysql_mutex_init(mrn_query_log_mutex)` is failed.